doc/nrf/ug_radio_fem.rst

```diff
@@ -41,14 +58,49 @@ Before you add the devicetree node in your application, complete the following s
    See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
 #. Enable support for MPSL implementation in |NCS| by setting the :kconfig:`CONFIG_MPSL` Kconfig option to ``y``.
 
-.. _ug_radio_fem_nrf21540_gpio:
+.. _ug_radio_fem_direct_support:
 
-Adding support for nRF21540 in GPIO mode
-****************************************
+Direct support
+==============
+
+If your application cannot use MPSL or if the FEM driver in MPSL does not support all features that you need, you can implement your own driver for nRF21540.
+The following samples use a direct FEM support:
+
+   * :ref:`direct_test_mode`
+   * :ref:`radio_test`
+
+The implementations supported by the above samples are the following:
+
+   * :ref:`ug_radio_fem_nrf21540_spi_gpio` - For the nRF21540 SPI/GPIO implementation that uses nRF21540.
```

Comment:
The SPI implementation is not existing right now. This should be reflected in the docs.

Reply:
For MPSL it isn/t existing but for DTM and radio test it exists

Reply:
That is true, however the file `doc/nrf/ug_radio_fem.rst` refers to the 'global' scope of FEM support. Please correct me if I'm wrong.

Reply:
Yes but under the MPSL and under the Direct Support header are explains which modes are supported with links to the corresponding sections